### PR TITLE
Update browser-features.md

### PR DESCRIPTION
**Pull request by vchapel**
Review and suggest changes or approve.

### CODE REVIEW

File: microsoft-edge/webview2/concepts/browser-features.md [hunk 1]
```diff
@@ -51,17 +51,24 @@ The following table displays the WebView2 features that differ from the Microsof
...
 | Translate | Off | No | This feature is turned off.  |  
 | Tracking Prevention | On`*` | No | `*` The UI for this feature has been removed, however the underlying functionality is still available.  Tracking prevention is always set to balanced.|  
-| Profile and Identity | Off | No | The feature that syncs your favorites, cookies, and so on, is turned off.  |  
-| Web Payment API | Off | No | This feature is turned off.  | 
+| Profile and Identity | Off | No | The feature that syncs your favorites, cookies, and so on, is turned off.  | 
 | Windows Defender Application Guard | Off | No | This feature is turned off.  |  
 | edge:// URLs | Review Details | No | Settings for the Microsoft Edge browser are on `edge://` URLs.  Because most of these webpages have Microsoft Edge branding or don't make sense within the context of WebView2, some of these URLs are turned off.  For more information, navigate to [Blocked internal URLs](#blocked-internal-urls).  |  
 
+## Platform Features
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/browser-features.md:63]:
```suggestion
## Web platform features
```

File: microsoft-edge/webview2/concepts/browser-features.md [hunk 2]
```diff
@@ -51,17 +51,24 @@ The following table displays the WebView2 features that differ from the Microsof
...
+
+The following table displays the WebView2 platform features that are currently removed.
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/browser-features.md:65]:
```suggestion
The following table displays the WebView2 web platform features that are currently unavailable.
```

File: microsoft-edge/webview2/concepts/browser-features.md [hunk 3]
```diff
@@ -51,17 +51,24 @@ The following table displays the WebView2 features that differ from the Microsof
...
+
+| Feature | Default state | Configurable | Details |  
+|:--- |:--- |:--- | :--- |  
+| Push Notifications | Off | No | This feature is not implemented in WebView2.  For more information, navigate to [Add support for HTML5 Notification API (#308)][GithubMicrosoftedgeWebview2feedbackIssues308]. |  
+| Web Payment API | Off | No | This feature is turned off. |
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/browser-features.md:70]: I suggest removing columns 2 and 3.

File: microsoft-edge/webview2/concepts/browser-features.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ In the context of WebView2, browser features adhere to the following design guid
 *   Most features work the same in WebView2 and Microsoft Edge.  If a feature doesn't make sense in the context of WebView2 or for other reasons, the feature is modified or turned off. 
 *   WebView2 features don't include Microsoft Edge branding.  
     
-## Features  
+## Browser Features
```
Comment (Reezaali) [inline microsoft-edge/webview2/concepts/browser-features.md:28]:
```suggestion
## Browser features  
```

**Review by Reezaali — CHANGES_REQUESTED** [review]

File: microsoft-edge/webview2/concepts/browser-features.md (hunk 1 above)
Comment (vchapel) [reply microsoft-edge/webview2/concepts/browser-features.md]: done

File: microsoft-edge/webview2/concepts/browser-features.md (hunk 2 above)
Comment (vchapel) [reply microsoft-edge/webview2/concepts/browser-features.md]: done

File: microsoft-edge/webview2/concepts/browser-features.md (hunk 3 above)
Comment (vchapel) [reply microsoft-edge/webview2/concepts/browser-features.md]: done

File: microsoft-edge/webview2/concepts/browser-features.md (hunk 4 above)
Comment (vchapel) [reply microsoft-edge/webview2/concepts/browser-features.md]: done

**Review by jasonstephen15 — APPROVED** [review]

**Review by Reezaali — APPROVED** [review]

**Pull request merged**